src/common/src/types/sentinel.rs

```diff
@@ -13,17 +13,17 @@
 // limitations under the License.
 
 use enum_as_inner::EnumAsInner;
-use risingwave_common::estimate_size::EstimateSize;
-use risingwave_expr::window_function::StateKey;
+
+use crate::estimate_size::EstimateSize;
 
 #[derive(Debug, Clone, PartialEq, Eq, EnumAsInner)]
-pub(super) enum KeyWithSentinel<T> {
+pub enum Sentinelled<T> {
```

Comment:
Can you add some doc comments for it? It's not clear what it is.

Reply:
Added.